### PR TITLE
Port of Wiki Documentation to Website

### PR DESCRIPTION
**Pull request by ConradWeiser**
Please refer to the preview. A section has been added for developer docs. 
This PR is not ready to be merged, as the documentation enclosed within needs updated.

Looking for help getting the documentation up to spec

### CODE REVIEW

**Review by ConradWeiser — COMMENTED** [review]
For now, I think this is ready to go! We'll fine-tune as 0.6 comes out, providing new features and changing requirements slightly.

File: README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ Pull requests are welcome. For major changes, please open an issue first to disc
 <div>
 	<p align="center">
 		<a href="https://github.com/Paperback-iOS/app/">
-			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=app&bg_color=0000&text_color=777&hide_border=true" alt="iOS App">
+			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=app&bg_color=0000&text_color=777&hide_border=true" alt="Android App">
```
Comment (FramboisePi) [inline README.md:24]:
```suggestion
			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=app&bg_color=0000&text_color=777&hide_border=true" alt="iOS App">
```

File: README.md [hunk 2]
```diff
@@ -21,10 +21,10 @@ Pull requests are welcome. For major changes, please open an issue first to disc
...
 		</a>
 		<a href="https://github.com/Paperback-iOS/extensions-common/">
-			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=extensions-common&bg_color=0000&text_color=777&hide_border=true" alt="Extensions Common">
+			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=extensions-common&bg_color=0000&text_color=777&hide_border=true" alt="App Extensions">
```
Comment (FramboisePi) [inline README.md:27]:
```suggestion
			<img src="https://github-readme-stats.vercel.app/api/pin/?username=Paperback-iOS&repo=extensions-common&bg_color=0000&text_color=777&hide_border=true" alt="Extensions Common">
```

**Review by FramboisePi — CHANGES_REQUESTED** [review]
I think this regression came from Soitora's branch issues.

**Review by FramboisePi — APPROVED** [review]
👍 
I agree with the idea. Looks good to me. Some small changes can be addressed latter.

**Pull request merged**